commit cfebbddea263fcb0ea8da01af611c0f4ce85ce79
Author: Phil Elwell <phil@raspberrypi.org>
Date:   Fri Mar 17 12:24:41 2017 +0000

    config: Make spidev a loadable module
    
    spidev isn't required early in the boot process, and not all users
    need it (spi_bcm2835 is a module), so make it a loadable module.
    
    See: https://github.com/raspberrypi/linux/issues/1897
    
    Signed-off-by: Phil Elwell <phil@raspberrypi.org>

diff --git a/arch/arm/configs/bcm2709_defconfig b/arch/arm/configs/bcm2709_defconfig
index b6ef78d6af1b..8198d4f73e23 100644
--- a/arch/arm/configs/bcm2709_defconfig
+++ b/arch/arm/configs/bcm2709_defconfig
@@ -615,7 +615,7 @@ CONFIG_I2C_GPIO=m
 CONFIG_SPI=y
 CONFIG_SPI_BCM2835=m
 CONFIG_SPI_BCM2835AUX=m
-CONFIG_SPI_SPIDEV=y
+CONFIG_SPI_SPIDEV=m
 CONFIG_PPS=m
 CONFIG_PPS_CLIENT_LDISC=m
 CONFIG_PPS_CLIENT_GPIO=m
diff --git a/arch/arm/configs/bcmrpi_defconfig b/arch/arm/configs/bcmrpi_defconfig
index f2e0a58a96c8..9eb7084f440c 100644
--- a/arch/arm/configs/bcmrpi_defconfig
+++ b/arch/arm/configs/bcmrpi_defconfig
@@ -611,7 +611,7 @@ CONFIG_I2C_GPIO=m
 CONFIG_SPI=y
 CONFIG_SPI_BCM2835=m
 CONFIG_SPI_BCM2835AUX=m
-CONFIG_SPI_SPIDEV=y
+CONFIG_SPI_SPIDEV=m
 CONFIG_PPS=m
 CONFIG_PPS_CLIENT_LDISC=m
 CONFIG_PPS_CLIENT_GPIO=m
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1119_cfebbddea263fcb0ea8da01af611c0f4ce85ce79.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1119_cfebbddea263fcb0ea8da01af611c0f4ce85ce79.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
